### PR TITLE
feat: add check version

### PR DESCRIPTION
**Pull request by tony-go**
Hi 👋 

This is a brand new PR (based on our recent updates) for issue #69 🎉 !

TODO:
- [x] add a `scannerVersion` property in the analysis object
- [x] add a summary process test (e2e)

### CODE REVIEW

**Comment by tony-go** [comment]
As you mentioned in the old PR @fraxken 

> use `scannerVersion` maybe instead of `version`

File: test/commands/summary.spec.js [hunk 1]
```diff
@@ -15,20 +15,19 @@ test("summary should execute summary command on fixtures 'result-test1.json'", a
...
     /.*/,
     /Extensions:.*$/,
-    /\(48\) {2}- \(50\) \.md - \(50\) \.json - \(50\) \.js - \(5\) \.ts - \(2\) \.yml.*$/,
+    /\(48\) {2}- \(50\) \.md - \(50\) \.js - \(50\) \.json - \(2\) \.yml - \(5\) \.ts.*$/,
     /.*/,
     /Licenses:.*$/,
     /\(47\) MIT - \(2\) ISC.*$/,
     /.*/
   ];
-  tape.plan(lines.length * 2);
```
Comment (tony-go) [inline test/commands/summary.spec.js:29]: It's complicated to keep `tape.plan` in these tests as the number of lines differs regarding the node version. 

Another approach could be to create a helper that asserts that a set of lines exists in the stream regardless of the number of lines.

**Comment by tony-go** [comment]
@fraxken Friendly ping!

File: src/commands/checkVersion.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import semver from "semver";
+
+export const kVersionRange = ">=0.7.0";
```
Comment (fraxken) [inline src/commands/checkVersion.js:3]: `>=3.0.0` here.

**Review by fraxken — APPROVED** [review]

**Pull request closed without merging**